access/grpc/grpc.go

```diff
@@ -1296,3 +1296,123 @@ func receiveBlocksFromClient[Client interface {
 		}
 	}
 }
+
+func (c *BaseClient) SubscribeBlockDigestsFromStartBlockID(
+	ctx context.Context,
+	startBlockID flow.Identifier,
+	blockStatus flow.BlockStatus,
+	opts ...grpc.CallOption,
+) (<-chan flow.BlockDigest, <-chan error, error) {
+	request := &access.SubscribeBlockDigestsFromStartBlockIDRequest{
+		StartBlockId: startBlockID.Bytes(),
+		BlockStatus:  convert.BlockStatusToEntity(blockStatus),
```

Comment:
These should only allow finalized or sealed as the status. otherwise, return an error